src/main/kotlin/no/nav/syfo/melding/domain/MeldingFraBehandler.kt

```diff
@@ -33,3 +34,10 @@ fun MeldingFraBehandler.toPMelding() =
         tekst = tekst,
         antallVedlegg = antallVedlegg,
     )
+
+fun MeldingFraBehandler.toMelding(behandlerRef: UUID) = Melding(
+    behandlerRef = behandlerRef, // TODO: Her må vi nok ha en behandlerPersonident -> behandlerRef mapping-funksjon
+    tekst = tekst ?: "",
+    tidspunkt = mottattTidspunkt,
+    innkommende = true,
+)
```

Comment:
Har ikke `behandlerRef` for `MeldingFraBehandler`, så har foreløpig tatt den inn som en hack for å få det til å kompilere.